src/common/low_precision_transformations/tests/pull_reshape_through_dequantization_transformation.cpp

```diff
@@ -91,7 +91,7 @@ class PullReshapeThroughDequantizationTransformation :
         const std::vector<ngraph::element::Type> supportedTypes = { ngraph::element::i8, ngraph::element::u8 };
         decomp->add_matcher<ngraph::pass::low_precision::PullReshapeThroughDequantization>(supportedTypes);
         decomp->add_matcher<ngraph::pass::low_precision::PullTransposeThroughDequantization>(supportedTypes);
```

Comment:
Why this and above are not changed to `ov::pass`?

Reply:
`src/common/low_precision_transformations` component is not in scope of this change. It only affects `src/common/transformations`.